### PR TITLE
new/list => list()

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Old syntax.
## Why It's Good For The Game

 It's better list inits are consistent.
## Changelog
:cl:
code: Converted instances of new/list to the list wrapper
/:cl:

### CODE REVIEW

File: code/game/objects/machinery/computer/security.dm [hunk 1]
```diff
@@ -280,7 +280,7 @@ What a mess.*/
 				var/temptext = R.fields["name"] + " " + R.fields["id"] + " " + R.fields["fingerprint"] + " " + R.fields["rank"]
 				for(var/i = 1, length(i<=components), i++)
 					if(findtext(temptext,components[i]))
-						var/prelist = new/list(2)
+						var/prelist = list(2)
```
Comment (Kapu1178) [inline code/game/objects/machinery/computer/security.dm:283]: these are not equivalent

File: code/game/objects/machinery/computer/skills.dm [hunk 2]
```diff
@@ -213,7 +213,7 @@ What a mess.*/
 				var/temptext = R.fields["name"] + " " + R.fields["id"] + " " + R.fields["fingerprint"] + " " + R.fields["rank"]
 				for(var/i = 1, length(i<=components), i++)
 					if(findtext(temptext,components[i]))
-						var/prelist = new/list(2)
+						var/prelist = list(2)
```
Comment (Kapu1178) [inline code/game/objects/machinery/computer/skills.dm:216]: these are not equivalent

**Review by Kapu1178 — CHANGES_REQUESTED** [review]
new /list(num) populates the list with that many nulls.

File: code/game/objects/machinery/computer/security.dm (hunk 1 above)
Comment (ivanmixo) [reply code/game/objects/machinery/computer/security.dm]: Thanks for pointing this out

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**